modules/diskq/Makefile.am

```diff
@@ -22,15 +14,43 @@ modules_diskq_libdisk_buffer_la_SOURCES = \
   modules/diskq/qdisk.h \
   modules/diskq/qdisk.c
 
+modules_diskq_libsyslog_ng_disk_buffer_la_CFLAGS	=	\
+	$(AM_CFLAGS) -fPIC
+modules_diskq_libsyslog_ng_disk_buffer_la_LIBADD	=	\
+	$(MODULE_DEPS_LIBS)
+modules_diskq_libsyslog_ng_disk_buffer_la_DEPENDENCIES	=	\
+	$(MODULE_DEPS_LIBS)
+
+modules_diskq_libdisk_buffer_la_SOURCES = \
+  modules/diskq/diskq.c \
+  modules/diskq/diskq.h \
+  modules/diskq/diskq-grammar.y \
+  modules/diskq/diskq-parser.c \
+  modules/diskq/diskq-parser.h \
+  modules/diskq/diskq-plugin.c \
+  $(modules_diskq_libsyslog_ng_disk_buffer_la_SOURCES)
+
+BUILT_SOURCES += \
+  modules/diskq/diskq-grammar.y \
+  modules/diskq/diskq-grammar.c \
+  modules/diskq/diskq-grammar.h
+
+EXTRA_DIST += $(BUILT_SOURCES) modules/diskq/diskq-grammar.ym
+
```

Comment:
By the way, I haven't seen $(BUILT_SOURCES) on any such right hand side before, so it's also probably an issue to deal with.

`git grep BUILT_SOURCES`